string command
string filename
int32 camera_number
bool write_file

